webapp/src/Controller/API/OrganizationController.php

```diff
@@ -42,8 +42,7 @@ public function __construct(
         ConfigurationService $config,
         EventLogService $eventLogService,
         AssetUpdateService $assetUpdater
-    )
-    {
+    ) {
```

Comment:
This violates opening a function brace on a newline...

Reply:
I think that is indeed fixed in my last PR, if/when that one is accepted I'll verify if this is still the case and otherwise see why phpcs doesn't detect this.